### ISSUE TITLE
dataset "corpus/imdb/imdb.train.txt.ss" and "train_both.pt"

### ISSUE THREAD

**Issue by drunkprogrammer** [yoyo-yun/MA-Bert#2]
hello, after I unzip the raw dataset from Duyu Tang and run it, it occurs error:
[Errno 2] No such file or directory: 'corpus/imdb/train_both.pt
But I don't find this file

**Comment by yoyo-yun** [comment]
Please check the right dataset paths. The file of train_both.pt would be generated automatically. If still errors, you could generate them by running function save_datasets() of common/utils.py.

**Comment by drunkprogrammer** [comment]
I try it again and I make sure the raw data is in the "corpus" directory. But I still don't know why it fails. The save_datasets()  function of utils.py doesn't have the argument to pass the data path and I am confused about the previous code.
"""
    from transformers import BertTokenizer
    pretrained_weights = 'bert-base-uncased'
    tokenizer = BertTokenizer.from_pretrained(pretrained_weights)
    processor = DATASET_MAP[config.dataset]()
    train_examples, dev_examples, test_examples = processor.get_documents()
"""
And class IMDB init data path is "imdb.train.txt.ss", the raw data in Duyu Tang is "train.txt", whether they are same?

If you are convenient, could you communicate with me?

Thanks a lot.

**Comment by Junzizaiji** [comment]
> I try it again and I make sure the raw data is in the "corpus" directory. But I still don't know why it fails. The save_datasets() function of utils.py doesn't have the argument to pass the data path and I am confused about the previous code. """ from transformers import BertTokenizer pretrained_weights = 'bert-base-uncased' tokenizer = BertTokenizer.from_pretrained(pretrained_weights) processor = DATASET_MAPconfig.dataset train_examples, dev_examples, test_examples = processor.get_documents() """ And class IMDB init data path is "imdb.train.txt.ss", the raw data in Duyu Tang is "train.txt", whether they are same?
> 
> If you are convenient, could you communicate with me on WeChat( ID: xiaoyue125125)?
> 
> Thanks a lot.

Hello, do you still keep the dataset, the author dataset link is broken, if it is convenient, can you send me the dataset you downloaded, thank you！！！

**Comment by drunkprogrammer** [comment]
Yes, I keep the dataset. The link is **https://pan.baidu.com/s/1iewWK8ZKgxsQQg497lGvBQ?pwd=ca9r**
Password：**ca9r** .

**Comment by Junzizaiji** [comment]
> Yes, I keep the dataset. The link is **https://pan.baidu.com/s/1iewWK8ZKgxsQQg497lGvBQ?pwd=ca9r** Password：**ca9r** .

Thank you very much！！！！！！！！！！！！

**Comment by 96mehakmohammad** [comment]
> Yes, I keep the dataset. The link is **https://pan.baidu.com/s/1iewWK8ZKgxsQQg497lGvBQ?pwd=ca9r** Password：**ca9r** .

Were you able to get the train_both.pt fixed issue fixed?

**Comment by 96mehakmohammad** [comment]
> Please check the right dataset paths. The file of train_both.pt would be generated automatically. If still errors, you could generate them by running function save_datasets() of common/utils.py.

Can you please please help me with the same issue, can't seem to figure out.
